packages/headless/src/features/facets/range-facets/generic/range-facet-reducers.test.ts

```diff
@@ -219,68 +219,44 @@ describe('range facet reducers', () => {
       expect(value.state).toBe('idle');
     });
 
-    it(`when #generateAutomaticRanges is false,
-    when a range in the payload is not found, it ignores it`, () => {
+    it('when a range in the payload is not found, it adds it to #currentValues', () => {
       const id = 'size';
-      const request = buildMockNumericFacetRequest({
-        generateAutomaticRanges: false,
-        currentValues: [],
-      });
-
+      const request = buildMockNumericFacetRequest({currentValues: []});
       state = {[id]: request};
 
       const value = buildMockNumericFacetValue();
       const nf = {[id]: [value]};
 
       handleRangeFacetSearchParameterRestoration(state, nf);
-      expect(request.currentValues).toEqual([]);
+      expect(request.currentValues).toContain(value);
     });
 
-    describe('when #generateAutomaticRanges is true', () => {
```

Comment:
Removed the `generateAutomaticRanges=true` restriction to restoring ranges not found in current values.